src/cml.js

```diff
@@ -220,11 +219,13 @@ class CML {
     };
 
     if (publish) {
-      report = String(
+      report = (
         await remark()
           .use(() => publishLocalFiles)
           .process(report)
-      );
+      )
+        .toString()
+        .replace(/\\&(.+)=/g, '&$1=');
```

Comment:
I am unfamiliar with syntax for referencing of the capture groups so assuming this is correct that

`\&something=` would be `$something=`

then +1